bots/tests-scan

```diff
@@ -139,23 +132,26 @@ def main():
 
     api = github.GitHub(repo=opts.repo)
 
+    # This options flow around the code, for example repo from opts is needed in
```

Comment:
The meaning of this is not so clear: "This options flow around the code".

How about "The repo option is used throughout the code".

Reply:
and perhaps prefix with a `#HACK:`, as using `api.repo` would be cleaner.